src/metrics/definitions.md

```diff
@@ -0,0 +1,9 @@
+# Metric Definitions
+
+It is important to understand that metrics are often defined in terms of a usage criterion.
+
+* A `usage criterion` defines what a metric is measuring.  It generally specifies a product and what specific use of that product qualifies as active.  For example, “Any Fenix Activity” measures activity for all Fenix profiles that have sent a `baseline` ping.  Similarly, “Opened DevTools” means that we measure activity for all desktop Firefox profiles that have sent a telemetry ping indicating that DevTools was opened.
+
+* A `dimension` allows slicing to a subset of profiles according to characteristics of those profiles.  Available dimensions include: country, channel, OS, etc.  A slice is a set of particular values within dimensions, for example, “country is US” is one slice and “country is either US or DE and channel is release” is another.
```

Comment:
Probably need to emphasise the word "slice" in the same way that we emphasise "dimension", "metric" and "usage criterion".

Reply:
Done.